src/tools/rustfmt/Cargo.toml

```diff
@@ -36,7 +36,7 @@ generic-simd = ["bytecount/generic-simd"]
 annotate-snippets = { version = "0.9", features = ["color"] }
 anyhow = "1.0"
 bytecount = "0.6.4"
-cargo_metadata = "0.15.4"
+cargo_metadata = "0.18"
```

Comment:
Can this wait until the next rustfmt sync?  https://github.com/rust-lang/rustfmt/pull/6034 already made this change.